nupic/__init__.py

```diff
@@ -19,3 +19,7 @@
 # http://numenta.org/licenses/
 # ----------------------------------------------------------------------
 __version__ = "0.3.0.dev0"
+
+import os
+NUPIC_ROOT = os.environ.get('NUPIC', os.path.dirname(os.path.realpath(__file__)))
```

Comment:
Let's wait what Matthew thinks ;) keeping it might be better for sidemodules or community projects that may rely on it. There are a lot and Matthew probably has the best overview. How about just a try-except implementation? 

<div>-------- Original message --------</div><div>From: Pascal Ehlert <notifications@github.com> </div><div>Date:03/30/2015  15:52  (GMT+01:00) </div><div>To: numenta/nupic <nupic@noreply.github.com> </div><div>Cc: Pascal Weinberger <passiweinberger@gmail.com> </div><div>Subject: Re: [nupic] Remove dependency from $NUPIC environment variable. Fixes
  numenta/nupic#1815, numenta/nupic#1947 (#1970) </div><div>
</div>In nupic/__init__.py:

> @@ -19,3 +19,7 @@
>  # http://numenta.org/licenses/
>  # ----------------------------------------------------------------------
>  __version__ = "0.3.0.dev0"
> +
> +import os
> +NUPIC_ROOT = os.environ.get('NUPIC', os.path.dirname(os.path.realpath(__file__)))
Hi David, my change will consider $NUPIC if it is set or consider the path from __file__ as you suggested. I didn't remove $NUPIC altogether because I wanted to avoid side-effects. If you guys are sure nobody relies on it, I'm more than happy to remove it for good. 

—
Reply to this email directly or view it on GitHub.